proposals/dao/tip_111.ts

```diff
@@ -0,0 +1,186 @@
+  await time.increase(timeRemaining);
+};
+
+// Tears down any changes made in setup() that need to be
+// cleaned up before doing any validation checks.
+const teardown: TeardownUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  console.log(`No actions to complete in teardown for fip${fipNumber}`);
+};
+
+// Run any validations required on the fip using mocha or console logging
+// IE check balances, check state of contracts, etc.
+const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  const daoSigner: SignerWithAddress = await getImpersonatedSigner(contracts.feiDAOTimelock.address);
+  await forceEth(contracts.feiDAOTimelock.address);
+
+  // check the stETH oracle price
+  const stethPerEth: number = (await contracts.compositeStEthEthOracle.read())[0] / 1e18;
+  expect(stethPerEth).to.be.at.least(0.9);
+  expect(stethPerEth).to.be.at.most(1);
+
+  // check the token movements
+  const aaveBalanceAfter: any = await contracts.aaveEthPCVDeposit.balance();
+  const balancerBalanceAfter: any = await contracts.balancerDepositFeiWeth.balance();
+  const aaveBalanceIncrease: any = aaveBalanceAfter.sub(aaveBalanceBefore) / 1e18;
+  const balancerBalanceDecrease: any = balancerBalanceBefore.sub(balancerBalanceAfter) / 1e18;
+  const ratio: any =
+    aaveBalanceIncrease > balancerBalanceDecrease
+      ? aaveBalanceIncrease / balancerBalanceDecrease
+      : balancerBalanceDecrease / aaveBalanceIncrease;
+  expect(ratio).to.be.at.least(1);
+  expect(ratio).to.be.at.most(1.001);
+
+  // check the new contract's state
+  expect(await contracts.ethLidoPCVDeposit.oracle()).to.be.equal(contracts.compositeStEthEthOracle.address);
+  expect(await contracts.ethLidoPCVDeposit.maximumSlippageBasisPoints()).to.be.equal('100');
+
+  // check the withdrawal from steth PCVDeposit
+  const stethBalanceBefore: any = (await contracts.steth.balanceOf(contracts.ethLidoPCVDeposit.address)) / 1e18;
+  const ethBalanceBefore: any = Number((await balance.current(addresses.aaveEthPCVDeposit)).toString()) / 1e18;
+  await contracts.ethLidoPCVDeposit.connect(daoSigner).withdraw(addresses.aaveEthPCVDeposit, '1000000000000000000');
+  const stethBalanceAfter: any = (await contracts.steth.balanceOf(contracts.ethLidoPCVDeposit.address)) / 1e18;
+  const ethBalanceAfter: any = Number((await balance.current(addresses.aaveEthPCVDeposit)).toString()) / 1e18;
+  const stethSpent: any = stethBalanceBefore - stethBalanceAfter;
+  const ethReceived: any = ethBalanceAfter - ethBalanceBefore;
+  expect(stethSpent).to.be.at.least(0.999);
+  expect(stethSpent).to.be.at.most(1.001);
+  expect(ethReceived).to.be.at.least(0.9);
+  expect(ethReceived).to.be.at.most(1);
+  expect(stethBalanceBefore).to.be.at.least(49000);
+  expect(stethBalanceBefore).to.be.at.most(51000);
+
+  // --------------------------------------------------------------
+  // FIP_104b validation
```

Comment:
nit: FIP_104 incorrect

Reply:
these are actually the steps from Tom's fip_104, that have been combined with tip_111 to not need a 2nd dao vote

Reply:
I called it FIP_104b to indicate it was the second part/cleanup (FIP_104 started the LBP, transferred CREAM and did various technical maintenance tasks)